frontend/packages/client/src/pages/ProposalCreate.js

```diff
@@ -55,6 +60,14 @@ export default function ProposalCreatePage() {
       return;
     }
 
+    if (!communityId) {
+      notifyError({
+        status: "No community information provided",
+        statusText:
+          "Please restart the proposal creation from the community page",
+      });
+      return;
+    }
```

Comment:
if at the end of the wizard the communityId query param is not in the Url, the proposal won't be created and an error message will be shown.